apps/stress-test/webpack.config.js

```diff
@@ -0,0 +1,130 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const TerserWebpackPlugin = require('terser-webpack-plugin');
+const { GriffelCSSExtractionPlugin } = require('@griffel/webpack-extraction-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+const isProd = process.env.NODE_ENV === 'production';
+
+const pages = [
+  {
+    name: 'v8-simple-stress',
+    path: './src/v8/simple-stress/index.tsx',
+    template: './src/v8/simple-stress/index.html',
+    output: 'v8/simple-stress/index.html',
+  },
+  {
+    name: 'v9-simple-stress',
+    path: './src/v9/simple-stress/index.tsx',
+    template: './src/v9/simple-stress/index.html',
+    output: 'v9/simple-stress/index.html',
+  },
+  {
+    name: 'wc-simple-stress',
+    path: './src/wc/simple-stress/index.wc.ts',
+    template: './src/wc/simple-stress/index.html',
+    output: 'wc/simple-stress/index.html',
+  },
+];
+
+const config = {
+  mode: isProd ? 'production' : 'development',
+  entry: pages.reduce((config, page) => {
+    config[page.name] = page.path;
+    return config;
+  }, {}),
+  output: {
+    filename: '[name].[contenthash].bundle.js',
+    sourceMapFilename: '[name].[contenthash].map',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devtool: 'source-map',
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|ts|tsx)$/,
+        // Apply "exclude" only if your dependencies **do not use** Griffel
+        // exclude: /node_modules/,
+        exclude: /\.wc\.(ts|tsx)?$/,
+        use: {
+          loader: GriffelCSSExtractionPlugin.loader,
+        },
+      },
+      // Add "@griffel/webpack-loader" if you use Griffel directly in your project
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: [/node_modules/, /\.wc\.(ts|tsx)?$/],
+        use: {
+          loader: '@griffel/webpack-loader',
+          options: {
+            babelOptions: {
+              plugins: ['@babel/plugin-proposal-class-static-block'],
+              presets: ['@babel/preset-typescript'],
+            },
+          },
+        },
+      },
+      // "css-loader" is required to handle produced CSS assets by Griffel
+      // you can use "style-loader" or "MiniCssExtractPlugin.loader" to handle CSS insertion
+      {
+        test: /\.css$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader'],
+      },
+
+      {
+        test: /\.(ts|tsx)?$/,
+        use: {
+          loader: 'babel-loader',
+        },
+        exclude: [/node_modules/, /\.wc\.(ts|tsx)?$/],
+      },
```

Comment:
As we have SWC in our repo already as a dependency of Nx, you can consider to use [`swc-loader`](https://swc.rs/docs/usage/swc-loader).

Reply:
I'm happy to use SWC here but I think we should make it a direct dependency rather than use it transitively via Nx. What do you think about adding `swc-loader` as a direct dependency?

Reply:
Yeah, you're right. Let's try to use `swc-loader` and add it as a direct dependency 👍

Reply:
Turns out Nx has `@swc/core` as a dependency but not `swc-loader` :D

I've added it for this app as the it's quite a lot faster.